src/lib/coinbase_receiver/coinbase_receiver.ml

```diff
@@ -0,0 +1,15 @@
+(* coinbase_receiver.ml *)
```

Comment:
I'm not sure this qualifies for its own toplevel library. Can you move this to `coda_base` or `consensus` instead?

Reply:
moved to `Consensus.Coinbase_receiver`